src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk

```diff
@@ -94,6 +94,10 @@
     <p>{{ t('{{ claimantName }} has rejected your defence.',
         { claimantName: claim.claimData.claimant.name }) }}</p>
     <p>{{ t('The court will review the case. We’ll email you if we set a hearing date to tell you how to prepare.') }}</p>
+    {% if claim.response.directionsQuestionnaire %}
+    <p>{{ t('They’ve also sent us their hearing requirements.') }}</p>
```

Comment:
can you fix the indentation here please? the `p` and the `endif` need aligning correctly
also you're using the same code in lots of places. Can you please extract it out to it's own utility macro. there are examples in the utilityMacro.njk file